src/vaults/harvests.utils.ts

```diff
@@ -188,9 +188,10 @@ async function evaluateYieldItems(
         const strategyInfo = await getStrategyInfo(chain, vault, { blockTag: item.block });
         const performanceScalar = 1 / (1 - strategyInfo.performanceFee / 10_000);
 
+        let eventDuration = duration;
         if (isInfluence && isIncentiveDistribution(vault, item)) {
           // TODO: codify this into influence configs or smth
-          duration = 14 * ONE_DAY_MS;
+          eventDuration = 14 * ONE_DAY_MS;
```

Comment:
when duration was overwritten here (although very unlikely) want to avoid having somehow a harvest and distro on the same timestamp causing the two weeks to be applied to multiple events